### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by obigriffith**
Reviewed text throughout site and made minor corrections and updates

### CODE REVIEW

File: text/faq.yml [hunk 1]
```diff
@@ -57,21 +57,15 @@ faq:
     - question: How are the gene and drug filters defined in DGIdb?
       anchor: filters
       answer: >
-        The filters currently include FDA-approved drugs, anti-neoplastic drugs, immunotherapy drugs, clinically actionable genes, 
-        genes included in the druggable genome definition and drug resistant genes. FDA approval status is extracted from ChEMBL. 
+        The filters currently include Approved drugs, Antineoplastic drugs, and Immunotherapies. FDA approval status is extracted from ChEMBL.
```
Comment (susannasiebert) [inline text/faq.yml:60]:
```suggestion
        The filters currently include Approved drugs, Antineoplastic drugs, and Immunotherapies. Regulatory approval status is extracted from ChEMBL. 
```

**Review by susannasiebert — CHANGES_REQUESTED** [review]
+1 Just one small correction. Otherwise this looks good.

File: text/contact.yml [hunk 2]
```diff
@@ -75,16 +75,16 @@ contact:
       people: Elaine Mardis, Rick Wilson
 
     - category: Additional software developers
-      people: Indraniel Das, James Koval, Nicholas Spies, Avinash Ramu, Jim Eldred
+      people: Indraniel Das, James Koval, Nicholas Spies, Avinash Ramu, Jim Eldred, Yang-Yang Feng, Jim Weible, Scott Smith, Ben Ainscough, Jonathan Song
```
Comment (ahwagner) [inline text/contact.yml:78]:
```suggestion
      people: Indraniel Das, James Koval, Nicholas Spies, Avinash Ramu, Jim Eldred, Yang-Yang Feng, Jim Weible, Scott Smith, Ben Ainscough
```

File: text/contact.yml [hunk 3]
```diff
@@ -75,16 +75,16 @@ contact:
...
     - category: Expert curators
-      people: Malachi Griffith, Obi Griffith, Alex Wagner, Janakiraman Subramanian, Nicholas Spies, Jaclyn Boozalis, Deng Pan
+      people: Malachi Griffith, Obi Griffith, Alex Wagner, Janakiraman Subramanian, Nicholas Spies, Jaclyn Boozalis, Deng Pan, Kelsy Cotto, Sharon Freshour
 
     - category: Additional conceptual contributors
       people: Timothy J. Ley, Li Ding, David J. Dooling, Ramaswamy Govindan
 
-    - category: Miscellaneous support
+    - category: Miscellaneous contributions
       people: Kilannin Krysiak, Zachary Skidmore, Robert Lesurf, Lee Trani
```
Comment (ahwagner) [inline text/contact.yml:90]:
```suggestion
      people: Kilannin Krysiak, Zachary Skidmore, Robert Lesurf, Lee Trani, Jonathan Song
```

**Review by ahwagner — CHANGES_REQUESTED** [review]
I agree with all of these changes, with one minor revision.

**Pull request merged**